capture/devconfig.py

```diff
@@ -116,4 +116,4 @@
 
 # must be 'default' or 'wolfram'
 # 'wolfram' is currently experimental and unsupported
-sampler = 'default'
+sampler = 'wolfram'
```

Comment:
Shouldn't default stay default?

Reply:
We pull wolfram in via requirements.  Everyone who is currently using is expecting wolfram... We might want to encode  fallback, but this was intentional.  I am actually MORE worried that they WON"T use wolfram.

Reply:
Like, if we run experiments using teh default sampler, that will be problematic.  We must hope that the center holds for wolfram.

Reply:
Not worried about MIT?

Reply:
Installing the wolframclient library in the requirements.txt does *not* install the wolfram kernel, which it needs to run.

Reply:
Let's leave default and I will just remember to tell the people what we are using.  I will also work on a code fix to write out which sampler is being used to the final observation interface (model and author columns)